src/ensembl/src/shared/components/loader/Loader.tsx

```diff
@@ -28,3 +28,33 @@ export const CircleLoader = (props: Props) => {
 
   return <div className={className} />;
 };
+
+export const ListLoader = (props: Props) => {
```

Comment:
Hmm, I wonder whether it isn't worth reorganising the `loader` folder like so:

```
└── loader
    ├── index.ts
    ├── CircleLoader.scss
    ├── CircleLoader.tsx
    ├── SidebarLoader.scss
    ├── SidebarLoader.tsx
```

so that different loaders are re-exported through the index file, but still live in their own files.

Also, I have a suspicion that the name ListLoader may be too broad — isn't this skeleton shape intended specifically for the sidebar?

Reply:
✅